pkg/api/v1/types.go

```diff
@@ -2030,6 +2030,9 @@ type SecurityContext struct {
 
 	// RunAsUser is the UID to run the entrypoint of the container process.
 	RunAsUser *int64 `json:"runAsUser,omitempty" description:"the user id that runs the first process in the container; see http://releases.k8s.io/HEAD/docs/design/security_context.md#security-context"`
+
+	// Run the container using the host ipc mode
+	UseHostIpc *bool `json:"useHostIpc,omitempty" description:"run the container using the host ipc mode; see http://releases.k8s.io/HEAD/docs/design/security_context.md#security-context"`
```

Comment:
I think we've gone back and forth a few times on having a pod level security context along with the container context so it's probably time to do it.  It should be easy to apply overrides if the kubelet first applies the pod context then the container context to the image run config.  

Will think on what this means in terms of policy and generating/validating an SC against allowed settings